doc/source/stack_and_queue.rst

```diff
@@ -20,3 +20,5 @@ Usage of Queue::
   x = front(q)
   x = back(q)
   x = dequeue!(q)
+
+Both ``Stack`` and ``Queue`` implement the Iterator interface; iterating over ``Stack`` returns items in FILO order and iterating over ``Queue`` returns items in FIFO order. There is also a ``reverse_iter`` function implemented for both which returns items in the reverse order for each type (i.e. FIFO for ``Stack`` and FILO for ``Queue``).
```

Comment:
heh yea I also thought this was strange, but the docs already were using `FILO` so I figured I would follow them here. I'll switch everything in this file to `LIFO`